lighthouse-core/audits/audit.js

```diff
@@ -216,6 +216,20 @@ class Audit {
     };
   }
 
+  /**
+   * @param {LH.Artifacts.NodeDetails} node
+   * @return {LH.Audit.Details.NodeValue}
+   */
+  static makeNodeValue(node) {
```

Comment:
this would be a bit more clear if it could be `makeNodeDetails`, but we took that name for the artifact type :/

`makeNodeItem`? `makeNodeItemValue`? `makeNodeTableItem`? Those would be true until we allow `node` details outside of tables, at least